### PR TITLE
[stable/chaoskube] #1899 add nodeSelector for chaoskube

### PR DESCRIPTION
**Pull request by linki**
[stable/chaoskube] Added nodeSelector for chaoskube.
related to https://github.com/kubernetes/charts/issues/1899

### CODE REVIEW

File: stable/chaoskube/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: chaoskube
-version: 0.6.1
+version: 0.6.1-1
```
Comment (linki) [inline stable/chaoskube/Chart.yaml]: Does anyone have a better approach? `v0.6.2` would be a bit confusing at this point.

Comment (mattfarina) [reply stable/chaoskube/Chart.yaml]: Why would 0.6.2 be confusing for a chart version? Charts are versioned separately from app versions. Adding the `-1` on the end notes that it's a prerelease to `0.6.1`. Helm expects semver.

Comment (linki) [reply stable/chaoskube/Chart.yaml]: @mattfarina Thanks for the hint. I changed it to 0.6.2.

**Pull request merged**